.github/pull_request_template.md

```diff
@@ -0,0 +1,24 @@
+<!-- Thank you for contributing to Pydantic AI! -->
+
+<!-- Please add the issue number that should be closed when this PR is merged. -->
+<!-- If there is no issue, please create one first, even for simple bug fixes. -->
```

Comment:
I suggest you drop text that is not necessary, the more text you have, the higher the chances that people will ignore it. I would.